### PR TITLE
Update to 3.3.3 LTS version

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,14 +1,14 @@
-val dottyVersion = "3.1.2" // dottyLatestNightlyBuild.get
+val dottyVersion = "3.3.3" // dottyLatestNightlyBuild.get
 
 lazy val root = project
   .in(file("."))
   .settings(
-    name := "dotty-xml-interpolator",
+    name := "scala-xml-interpolator",
```
Comment (hamzaremmal) [inline build.sbt:6]: Can we change the name of a project ?

File: build.sbt [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
     version := "0.1.0",
```
Comment (hamzaremmal) [inline build.sbt:7]: Oh, it's not a 1.x, I guess changing the name is not a problem then.

**Review by hamzaremmal — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (nicolasstucki) [reply build.sbt:6]: We haven't published any version yet

Comment (hamzaremmal) [reply build.sbt:6]: Right !

**Pull request merged**